api/src/pcapi/routes/backoffice/pro/blueprint.py

```diff
@@ -257,3 +267,133 @@ def create_offerer() -> utils.BackofficeResponse:
 
     flash(Markup("La structure et le lieu <b>{name}</b> ont été créés").format(name=venue.common_name), "success")
     return redirect(url_for("backoffice_web.offerer.get", offerer_id=user_offerer.offererId), code=303)
+
+
+class MessageException(Exception):
+    def __init__(self, message: str):
+        self.message = message
+        super()
+
+
+def get_user_id_for_connect_as_base_query() -> BaseQuery:
+    return (
+        users_models.User.query.with_entities(users_models.User.id)
+        .join(users_models.User.UserOfferers)
+        .filter(
+            users_models.User.isActive == True,
+            ~users_models.User.roles.contains([users_models.UserRole.ADMIN]),
+            ~users_models.User.roles.contains([users_models.UserRole.ANONYMIZED]),
+            users_models.User.roles.contains([users_models.UserRole.PRO]),
```

Comment:
Pourquoi ne pas utiliser les propriétés hybrides, comme on le fait en général ?
```suggestion
            ~users_models.User.has_admin_role,
            ~users_models.User.has_anonymized_role,
            users_models.User.has_pro_role,
```